cpp/tests/io/json_test.cpp

```diff
@@ -914,8 +914,7 @@ TEST_P(JsonReaderParamTest, InvalidFloatingPoint)
 
   const auto col_data = cudf::test::to_host<float>(result.tbl->view().column(0));
   // col_data.first contains the column data
-  for (const auto& elem : col_data.first)
-    ASSERT_TRUE(std::isnan(elem));
+  // ignore all data because it is all nulls.
   // col_data.second contains the bitmasks
   ASSERT_EQ(0u, col_data.second[0]);
```

Comment:
do we need to_host here?
I'd assume that we just need to check `num_rows()` and `null_count()` of `result.tbl->view().column(0)`.

Reply:
updated.